src/compiler/ast-node-typeini.bas

```diff
@@ -522,7 +522,15 @@ private sub hFlushExprStatic( byval n as ASTNODE ptr, byval basesym as FBSYMBOL
 	if( sym = NULL ) then
 		sym = basesym
 	end if
+
 	var sdtype = symbGetType( sym )
+	var sfulldtype = symbGetType( sym )
```

Comment:
`sfulldtype` has same value as `sdtype` throughout.

Reply:
oops; fixed it.